src/PythonMigrationViewExtension/IronPythonInfoDialog.xaml.cs

```diff
@@ -24,5 +70,56 @@ private void OnMoreInformationButtonClicked(object sender, RoutedEventArgs e)
             this.ViewModel.OpenPythonMigrationWarningDocumentation();
             this.Close();
         }
+
+        private void OnCustomNodeClick(object sender, RoutedEventArgs e)
+        {
+            if (UserDefinitionCustomNodesContainingPython.SelectedItems.Count == 1)
+            {
+                CustomNodeItem selectedCNItem = UserDefinitionCustomNodesContainingPython.SelectedItem as CustomNodeItem;
```

Comment:
could this ever be null after this `as`?

Reply:
I dont think it can be null here, as we are checking that the only one list item is selected. and the UserDefinitionCustomNodesContainingPython contains only user defined custom nodes.

Reply:
Do you think the OpenCustomNodeWorkspace should be called from try-catch?